src/video/windows/SDL_windowswindow.c

```diff
@@ -180,6 +188,25 @@ static DWORD GetWindowStyleEx(SDL_Window *window)
     return style;
 }
 
+#ifdef HAVE_SHOBJIDL_CORE_H
+static ITaskbarList3* GetTaskbarList(SDL_Window* window)
+{
+    const SDL_WindowData *data = window->internal;
+    if (data->videodata->taskbar_button_created && !data->videodata->taskbar_list) {
```

Comment:
Also need to set an error when `taskbar_button_created` is false. It's probably better to restructure this function with early outs and specific errors.